cfme/configure/configuration/__init__.py

```diff
@@ -1821,6 +1512,33 @@ def delete(self, cancel=True):
             sel.handle_alert()
 
 
+@navigator.register(Tag, 'All')
+class TagsAll(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance.server.zone.region', 'Details')
+
+    def step(self):
+        tabs.select_tab("My Company Tags")
+
+
+@navigator.register(Tag, 'Add')
+class TagsAdd(CFMENavigateStep):
+    """Unlike most other Add operations, this one requires an instance"""
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        fill(tag_form, {'category': self.obj.category.display_name})
```

Comment:
Not sure what you mean here, but to preserve the use of objects, you must pass a Category object to a Tag when you create it.